src/models/Quote/Quote.js

```diff
@@ -4,11 +4,11 @@ function Quote() {
 
     const formRecord = {
         inputs: { 
-            firstName: 'Eamonn',
-            lastName: '',
-            dob: '',
-            term: '',
-            email: ''
+            firstName: 'Kevin',
```

Comment:
Gone from indents of 2 spaces to indents of 4 spaces.

Reply:
this looks like its 2 spaces in my local. Not sure.